src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -133,6 +137,14 @@ public static ProjectFileInfo Create(MSBuildOptions options, ILogger logger, str
                 {
                     projectFileInfo.DefineConstants = defineConstants.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).Distinct().ToList();
                 }
+
+                var signAssembly = projectInstance.GetPropertyValue("SignAssembly");
+                if (!string.IsNullOrWhiteSpace(signAssembly))
+                {
+                    projectFileInfo.SignAssembly = Convert.ToBoolean(signAssembly);
```

Comment:
i see. well, then we can address similar issue together in another PR. i'm fine with this change now.